### PR TITLE
Feature/rewarded ad

### PR DESCRIPTION
**Pull request by lucasvdiepen**

### CODE REVIEW

**Review by PiterGroot — APPROVED** [review]
LGTM 👍

File: Assets/Scripts/Framework/Sharing/ShareHighscore.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
+using ProefExamen.Framework.UI;
 using ProefExamen.Framework.Utils;
```
Comment (Joost-Robat) [inline Assets/Scripts/Framework/Sharing/ShareHighscore.cs]: Move namespace imports according to code convecntions.

File: Assets/Scripts/Framework/UI/TextUpdater.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            foreach ((string key, string value) in _tags)
+                defaultText = defaultText.Replace(key, value);
+
+            _text.text = defaultText;
+        }
+
+        /// <summary>
+        /// Replaces the text of the default tag set in the inspector.
+        /// </summary>
+        /// <param name="text"></param>
```
Comment (Joost-Robat) [inline Assets/Scripts/Framework/UI/TextUpdater.cs]: Please enter summary for param "text".

File: Assets/Scripts/Framework/UI/TextUpdater.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        public void ReplaceTag(string text)
+        {
+            if(string.IsNullOrEmpty(_tag))
+            {
+                Debug.LogError("No tag set for TextUpdater");
+                return;
+            }
+
+            ReplaceTag(_tag, text);
+        }
```
Comment (Joost-Robat) [inline Assets/Scripts/Framework/UI/TextUpdater.cs:56]:
```suggestion
        public void ReplaceTag(string text)
        {
            if(string.IsNullOrEmpty(_tag))
                 Debug.LogError("No tag set for TextUpdater");
            else
                 ReplaceTag(_tag, text);
```

**Review by Joost-Robat — COMMENTED** [review]
Looking good, couple of irregularities. After applying feedback ready for merge.

File: Assets/Scripts/Framework/UI/TextUpdater.cs (hunk 3 above)
Comment (lucasvdiepen) [reply Assets/Scripts/Framework/UI/TextUpdater.cs:56]: Resolved

**Review by Joost-Robat — APPROVED** [review]
Looking good. Approved.

**Pull request merged**